apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/SoeskenjusteringUndertype.tsx

```diff
@@ -0,0 +1,91 @@
+import { IDetaljertBehandling } from '~shared/types/IDetaljertBehandling'
+import {
+  BarnepensjonSoeskenjusteringGrunn,
+  RevurderingInfo,
+  SOESKENJUSTERING_GRUNNER,
+  SoeskenjusteringInfo,
+  tekstSoeskenjustering,
+} from '~shared/types/RevurderingInfo'
+import { FormEvent, useState } from 'react'
+import { BodyShort, Button, Heading, Select } from '@navikt/ds-react'
+import { hentBehandlesFraStatus } from '~components/behandling/felles/utils'
+import { isPending, useApiCall } from '~shared/hooks/useApiCall'
+import { lagreRevurderingInfo } from '~shared/api/revurdering'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { oppdaterRevurderingInfo } from '~store/reducers/BehandlingReducer'
+import styled from 'styled-components'
+
+function hentUndertypeFraBehandling(behandling?: IDetaljertBehandling): SoeskenjusteringInfo | null {
+  const revurderinginfo = behandling?.revurderinginfo
+  if (revurderinginfo?.type == 'SOESKENJUSTERING') {
```

Comment:
Har vi ingen konstant for typen her?

Reply:
Jo, den er definert som en `type`, så string-konstanten er selve referansen. Gir mer mening når du navigerer i intellij f.eks.

Reply:
men ingen spesifikk "navngitt" konstant da